Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,469 @@
+The type of nozzle that will be tested is a minimum length supersonic nozzle. Since there is ambient pressure, the nozzle will be optmized for ambient conditions.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of this project is to create a cold gas thruster for on-ground experimentation. The goal for the data is to be able to design a nozzle for a small propulsion system, where boundary
+layer effects are dominent, but also to achieve an specific impulse of 50s. Using the thrust over time curves, the specific impulse can be
+ calculated numerically using MATLAB\@. The data will be transferred and analyzed there using the following equations as a basis.
+
+
+\begin{equation} \label{specific impulse}
+  I_t    = \int_{t_i}^{t_f} F_{thrust} dt
+\end{equation}
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/cold-gas-thruster.png}
+  \caption{This is the Solidworks Assembly of the current design of the SPEX cold gas thruster. The piping interface to the nozzle is omitted for clarity.}
+\label{fig:lifecycle}
+\end{figure}
+
+\begin{equation} \label{specific impulse}
+  I_s = \frac{I_t}{m_p g_0}
+\end{equation}
+
+As seen above, calculating the specific impulse is
+an easy matter if the data in reliable. It can be unreliable if the system does not perform well or if there is noise in the data. The main difficulty in collecting data in the accuracy of
+the load cell. The load cell that captured correct data in the Spring of 2017 was accurate within 5\%. This is accurate enough for our forces.
+Vacuum condition availablity is low at RIT, so the team will design the nozzles for ambient conditions.
+
+
+
+% \section{Secondary Objectives}
+% \label{sec:secondary-obj}
+% Secondary Objectives are lower priority or bonus objectives that are significant but not the main focus of the project. This template does not have secondary objectives.
+
+\section{Benefit to SPEX}
+\label{sec:benefit}
+% One of the core values of SPEX is to provide opportunities for academic and professional growth for its members,
+% and to challenge them with interesting projects.
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+
+This project would be benefical to SPEX because of the experience given to SPEX in propulsion system design and experimentation. After its completion, SPEX will have a cold gas thruster that
```

Comment:
We are really hoping to get into what "experience" actually entails. Learning about propulsion theory, practice with prototyping and testing designs, or building test benches... I'd like this to be a bit more detailed if you can. Doesn't have to be exhaustive, but spend a little time thinking about what "experience" actually means to you and the team.